examples/nlp/ipynb/parameter_efficient_finetuning_of_gpt2_with_lora.ipynb

```diff
@@ -42,7 +42,9 @@
     "[GPT-2 model](https://keras.io/api/keras_nlp/models/gpt2/) and fine-tune\n",
     "it on the next token prediction task using LoRA. We will compare LoRA GPT-2\n",
     "with a fully fine-tuned GPT-2 in terms of the quality of the generated text,\n",
-    "training time and GPU memory usage."
+    "training time and GPU memory usage.\n",
+    "\n",
+    "Note: This example runs on the TensorFlow backend, purely because TensorFlow"
```

Comment:
Please regenerate the notebook, it seems there was a mixup (same in the md file)

Reply:
done!